soapui/src/main/java/com/eviware/soapui/plugins/PluginManager.java

```diff
@@ -87,7 +87,16 @@ public boolean accept(File pathname) {
             }
         });
         if (pluginFiles != null) {
-            List<File> pluginFileList = Arrays.asList(pluginFiles);
+            List<File> pluginFileList = new ArrayList<>();
+            ProductBodyguard productBodyguard = new ProductBodyguard();
+            for (File f:pluginFiles) {
+                if (!productBodyguard.isKnown(f)) {
+                    SoapUI.log.warn("Plugin '" + f.getName() + "' is not loaded since it's not approved by SmartBear Software.");
```

Comment:
I think that the log message should be "because it hasn't been signed by" rather than "since it's not approved by". @mattihjelm, any thoughts?